pkg/scheduler/objects/queue.go

```diff
@@ -473,6 +473,9 @@ func (sq *Queue) GetPartitionQueueDAOInfo() dao.PartitionQueueDAOInfo {
 	} else {
 		queueInfo.Parent = sq.QueuePath[:strings.LastIndex(sq.QueuePath, configs.DOT)]
 	}
+	queueInfo.MaxRunningApps = sq.maxRunningApps
+	queueInfo.RunningApps = sq.runningApps
+	queueInfo.AllocatingAcceptedApps = make([]string, len(sq.allocatingAcceptedApps))
```

Comment:
Thanks for pointing out!

I updated to **queueInfo.AllocatingAcceptedApps = make([]string, 0)** since we don't know how many elements initially. 
Also I only **append appID when sq.allocatingAcceptedApps value is true** since we don't care about appID when it's false.